### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by naolkidanu**
## In this PR I have done the following tasks:
- Implement Add and Remove functionality
- Implement edit functionality
- Update all remaining items' indexes
- Enhance local storage functionality
- Checkout linter

### CODE REVIEW

**Review by fobadara — APPROVED** [review]
Hi @naolkidanu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Good first attempt

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by naolkidanu** [comment]
Hi@Fobadara, I appreciate you looking over and approving my project.

File: dist/index.html [hunk 1]
```diff
@@ -1,35 +1,29 @@
...
-        <input class="input" type="text" placeholder="Add to your list..." />
-        <button type="button" class="btn btn-add">
-          <i class="fa fa-plus-square" aria-hidden="true"></i>
-        </button>
-      </div>
-      <ul class="list"></ul>
-      <button type="button" class="btn btn-clear">Clear all completed</button>
+      <button type="button" class="btn-refresh">
+        <i class="fa fa-refresh" aria-hidden="true"></i>
+      </button>
```
Comment (naolkidanu) [inline dist/index.html:16]: Please place this button on the right corner of the heading instead if at the bottom of the heading: this makes it unfriendly.

**Review by naolkidanu — COMMENTED** [review]
Required Changes ♻️

**Pull request merged**